nextcloud-aio-helm-chart/readme.md

```diff
@@ -12,3 +25,13 @@ Then run:
 helm repo add nextcloud-aio https://nextcloud.github.io/all-in-one/
 helm install my-release nextcloud-aio/nextcloud-aio-helm-chart -f values.yaml
 ```
+
+And after a while, everything should be set up.
+
+## How to update?
+Since the values of this helm chart may change in the future, it is highly recommended to strictly follow the following procedure whenever you want to upgrade it.
+1. Stop all running pods
```

Comment:
@st3iny do you know if there a helm command that does this?

Reply:
Not that I know of. There is only `kubectl scale --replicas=0` which needs to run for every Deployment/StatefulSet.

Reply:
I see, then I'll leave it as is